crates/humanode-peer/src/chain_spec.rs

```diff
@@ -328,23 +340,33 @@ fn testnet_genesis(
         ethereum_chain_id: EthereumChainIdConfig {
             chain_id: ETH_CHAIN_ID,
         },
-        evm: Default::default(),
-        evm_accounts_mapping: EvmAccountsMappingConfig {
-            mappings: vec![
-                (
-                    account_id("Alice"),
-                    EthereumAddress(ETHEREUM_ADDRESS_GERALD),
-                ),
-                (account_id("Bob"), evm_account("Bob")),
-            ],
+        evm: EVMConfig {
+            accounts: {
+                let evm_pot_accounts = vec![(
+                    humanode_runtime::EvmToNativeSwapBridgePot::account_id(),
+                    evm_genesis_account(INITIAL_POT_ACCOUNT_BALANCE),
```

Comment:
This should be the existential balance + sum of all endowed native accounts excluding the bridge pot.

Reply:
https://github.com/humanode-network/humanode/pull/673/commits/0dda71e2cb28c04743c355dae590571cbfead814

Reply:
https://github.com/humanode-network/humanode/pull/673/commits/0c72e91188f014f347f75923cd07d1b37f2a9043

Reply:
Will be resolved as part of #695.